doc/classes/SpinBox.xml

```diff
@@ -13,23 +26,29 @@
 			<return type="LineEdit">
 			</return>
 			<description>
+				Returns the [LineEdit] id of the [SpinBox].
```

Comment:
It's not an id, but the actual LineEdit node instance used in the SpinBox (so that you can call LineEdit-specific methods on it).

Reply:
Isn't it an object ID of the line edit?